### DISCUSSION TITLE
인기글 배너 정책 다시 정하기

### DISCUSSION THREAD

**Discussion by jminkkk** [woowacourse-teams/2024-code-zap#912] [이야기해봐요]
# 인기글 배너

현재는 주요 태그들을 고정하여 템플릿 목록을 조회하고 있다.

프리코스가 끝난 지금 배너 정책에 대해 다시 논의를 해봅시다.

## 안건

0. 인기 태그별 템플릿을 조회 vs 인기 템플릿을 조회
1. 현재처럼 고정 태그 유지 vs 인기 태그로 조회
   + 겹치는 태그들을 어떻게 처리할 것인가?
   + 배너 목록 조회 시에 사진이 함께 보이도록 태그의 이미지를 추가할 것인가?
2. 배너 클릭 시 정렬을 최근순 등록순 vs 좋아요 순
   +  좋아요 순으로 정렬하게 될 경우 1페이지만 보일 것인가?

**Comment by zeus6768** [comment]
### 헤인
- 인기 태그를 진짜 인기 있는 태그로 조회하자. 
- 사진은 빠지게 될 것이다. 
- 색상만 들어간다. 
- 디자인 레퍼런스를 찾아봐야 한다. 
- 태그 설명도 빠지게 되려나...?

### 제우스
- 사용자 피드백에서 온 기획이 아니다. 
- 그러므로 원래대로 되돌려도 괜찮다. 

### 초롱
- 사용자에게 중요한 기능이 아니다. 
- 기술적 도전을 해도 된다. 
- 진짜 인기 게시글을 보여주자.

**Comment by jminkkk** [comment]
## 11월 14일 회의 진행 결론

### 현재처럼 고정 태그 유지 vs 인기 태그로 조회

실제 인기 태그를 조회하자

### 배너 목록 조회 시에 사진이 함께 보이도록 태그의 이미지를 추가할 것인가?

이미지를 삽입하지 않고 기존 배너에서 글과 색상만 남기도록 한다.

### 겹치는 태그들을 어떻게 처리할 것인가?

문제가 발생할 경우에 처리한다.

### 배너 클릭 시 정렬을 최근순 등록순 vs 좋아요 순

정렬은 현재 최근순으로 유지한다.

### 인기 태그 정책 결정

"지금 인기있는 태그"

- 7일 동안 생성된 템플릿 목록 중에서 가장 많이 연결된 태그를 조회한다.
- 태그는 5개씩 조회한다.
- 7일 동안의 태그가 5개 이하인 경우 기간은 연장될 수 있다.
